pages/doc/integrations_new_changed_2022.md

```diff
@@ -24,9 +24,9 @@ We made improvements to the following integrations in December 2022:
   
     ![Screenshot of the new Kubernetes setup UI experience.](images/k8s-setup-UI.png)
   
-  - We will deprecate the Helm or manually-installed Kubernetes Metrics Collector and Wavefront proxy next year. Our new Kubernetes Operator replaces the Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy for all Kubernetes Distributions except for OpenShift Container Platform. 
+  - We deprecate the Helm or manually-installed Kubernetes Metrics Collector and Wavefront proxy. Our new Kubernetes Operator replaces the Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy for all Kubernetes Distributions except for OpenShift Container Platform. 
```

Comment:
Our new Kubernetes Operator replaces the Helm or manually installed Kubernetes Metrics Collector and Wavefront proxy for all Kubernetes Distributions except for the OpenShift Container Platform.